ui/dasher/css/_dasher.scss

```diff
@@ -192,6 +192,10 @@
     }
   }
 
+  .piece .list {
+    overflow: auto;
```

Comment:
Useful if the screen height is insufficient. However the scroll bar puts the piece list on 2 columns instead of 3